ui/apps/platform/src/Containers/MainPage/Sidebar/NavigationContent.tsx

```diff
@@ -1,18 +1,20 @@
-import React, { ReactElement, ReactNode } from 'react';
+import React, { CSSProperties, ReactElement, ReactNode } from 'react';
 import { Label } from '@patternfly/react-core';
 
 import TechPreviewLabel from 'Components/PatternFly/TechPreviewLabel';
 
 type NavigationContentVariant = 'Deprecated' | 'TechPreview';
 
+const style = {
+    '--pf-v5-c-label__content--Color': 'var(--pf-v5-global--Color--dark-100)',
+} as CSSProperties;
+// Type assertion prevents TypeScript error:
+// error TS2353: Object literal may only specify known properties, and ''--pf-v5-c-label__content--Color'' does not exist in type 'Properties<string | number, string & {}>'.
+// The expected type comes from property 'style' which is declared here on type 'IntrinsicAttributes & LabelProps'
```

Comment:
Is it possible that something in the React types caused our [override here](https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/react-app-env.d.ts#L16) to fail? I wonder if there is an alternative we can use that still allows us to omit the type assertions when setting PatternFly CSS vars?

Reply:
Aha, good point. Looks like only 3 commits after 18.3

https://github.com/DefinitelyTyped/DefinitelyTyped/commits/master/types/react

Reply:
Okay, it looks like the following contribution opened the door:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67583

And the following revert closed the door:
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/79bae95b770c7209689eacc033d93ce9998f4816

Reply:
It was fun while it lasted 😭 

I can re-investigate at another point in time to see if we can still get this ability, but it isn't a high priority right now.